articles/active-directory/develop/tutorial-v2-windows-uwp.md

```diff
@@ -108,92 +111,95 @@ This section shows how to use MSAL to get a token for Microsoft Graph API. Make
     ```csharp
     public sealed partial class MainPage : Page
     {
-        //Set the API Endpoint to Graph 'me' endpoint
-        string graphAPIEndpoint = "https://graph.microsoft.com/v1.0/me";
-
+       
         //Set the scope for API call to user.read
         string[] scopes = new string[] { "user.read" };
 
-        // Below are the clientId (Application Id) of your app registration and the tenant information.
+        // Below are the clientId (Application Id) of your app registration and the tenant information. 
         // You have to replace:
         // - the content of ClientID with the Application Id for your app registration
         // - the content of Tenant with the information about the accounts allowed to sign in in your application:
         //   - for Work or School account in your org, use your tenant ID, or domain
         //   - for any Work or School accounts, use organizations
         //   - for any Work or School accounts, or Microsoft personal account, use common
         //   - for Microsoft Personal account, use consumers
-        private const string ClientId = "0b8b0665-bc13-4fdc-bd72-e0227b9fc011";
+        private const string ClientId = "[Enter client ID of the app as obtained from Azure Portal, e.g. 4a1aa1d5-c567-49d0-ad0b-cd957a47f842]";        
 
-        public IPublicClientApplication PublicClientApp { get; }
+        public IPublicClientApplication PublicClientApp { get; } 
 
         public MainPage()
         {
-          this.InitializeComponent();
+            this.InitializeComponent();
 
-          PublicClientApp = PublicClientApplicationBuilder.Create(ClientId)
+            PublicClientApp = PublicClientApplicationBuilder.Create(ClientId)
                 .WithAuthority(AadAuthorityAudience.AzureAdAndPersonalMicrosoftAccount)
                 .WithLogging((level, message, containsPii) =>
                 {
                     Debug.WriteLine($"MSAL: {level} {message} ");
                 }, LogLevel.Warning, enablePiiLogging:false,enableDefaultPlatformLogging:true)
-                .WithUseCorporateNetwork(true)
-                .Build();
+                .WithUseCorporateNetwork(false)
+                .WithRedirectUri(Constant.PublicClientRedirectUri)
```

Comment:
Use .WithDefaultRedirectUri() please

Reply:
Added Alternate approach to use WithDefaultRedirectURI().
Thanks!